lib/config.d.ts

```diff
@@ -0,0 +1,268 @@
+/// <reference types="node" />
+
+import * as http from 'http';
```

Comment:
Would doing `import {Agent as httpAgent} from 'http';` and `import {Agent as httpsAgent} from 'https';` make a difference in how much typings data is loaded into memory by Intellisense or by the typescript compiler? I've seen complaints about Intellisense slowing down VS Code or the typescript compiler being too slow when declaration files are large. Since the only typings we need are for `Agent` from both modules, it seems unnecessary to load all of the typings from those modules into memory.